### DISCUSSION TITLE
Table and Column creation with API

### DISCUSSION THREAD

**Discussion by awdDev786** [supabase/supabase#207] [Questions] [answered]
Hello!i want to use supabase totally with node API.But i cant see the options like database creation in a project,table creation in database and even column creation with using API.
Does this support mentioned functionality?

**Comment by soedirgo** [comment, marked as answer]
Hi @awdDev786!

We plan to expose this functionality through [pg-api](https://github.com/supabase/pg-api). It's not as polished as our supabase-js API at the moment, but we'll get there. :)

You can check out [this discussion](https://github.com/supabase/supabase/discussions/70) for more on this.
